spec/models/user_spec.rb

```diff
@@ -63,6 +63,8 @@
   describe "abilites" do
     let(:bob) { create(:bob) }
     let(:alice) { create(:alice) }
+    let(:bob_wishlist_item) { create(:item, wishlist: bob.wishlists.first) }
+    let(:alice_wishlist_item) { create(:item, wishlist: alice.wishlists.first) }
```

Comment:
You could populate their wishlists already in the factory. That's kind of the purpose of `bob` and `alice` They've got everything we need to test. So some wishlists items in their wishlists would be nice to have on default.